render/template.go

```diff
@@ -3,12 +3,12 @@ package render
 import (
 	"html/template"
 	"io"
-	"log"
 	"os"
 	"path/filepath"
 	"sort"
 	"strings"
 
+	"github.com/Sirupsen/logrus"
```

Comment:
The repo is now lower case `sirupsen`. Can you please undo these import changes?